.workleap.rules.yaml

```diff
@@ -16,7 +16,7 @@ rules:
   oas3-operation-security-defined: true
 
   schema-ids-must-have-alphanumeric-characters-only:
-    description: 'All schema ids must only contain alphanumeric characters.'
+    description: 'All schema ids must only contain alphanumeric characters. Refer to: https://gsoftdev.atlassian.net/wiki/spaces/TEC/pages/3858235678/IDP+OpenAPI+Rulesets#schema-ids-must-have-alphanumeric-characters-only'
```

Comment:
Do we have a "url shortener" system available? 
I don't care about the length of the link, but it would be better to have a link that never changes in the code, while being able to update the target if needed.

Reply:
It would be definitely valid, but we currently don't have a solution at workleap, it was requested through user voice though: https://github.com/gsoft-inc/wl-leap-uservoice/discussions/35